packages/react-app/src/pages/_app.tsx

```diff
@@ -34,7 +36,7 @@ function MyApp({ Component, pageProps: { session, ...pageProps }, router }: AppP
 		<SessionProvider session={session}>
 			<CustomerContext.Provider value={{ customer, setCustomer }}>
 				<ChakraProvider resetCSS theme={theme}>
-					<AuthContextProvider user={exampleUser}>
```

Comment:
Question about a comment you left in `useRoles`

```
   * This hook is where we can call an API and return the user's roles
	 * We will probably want to pass in the user at some stage, although currently
	 * we only use discord roles, so can use the getSession method on the server side
	 * Note, we currently only support roles in bankless, and need to establish a more complete
	 * Auth method for multitenancy
```
so when would pass in a user here?

Reply:
We will be deprecating user in the next release - good spot